aten/src/ATen/native/vulkan/ops/Packing.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <ATen/native/vulkan/ops/Common.h>
+
+namespace at {
+namespace native {
+namespace vulkan {
+namespace ops {
+
+vTensor pack_image2d_h2w2(vTensor v_src, api::Context* context, api::Command::Buffer& command_buffer);
+vTensor unpack_image2d_h2w2(vTensor v_src, c10::SmallVector<int64_t, 4u> output_sizes, api::Context* context, api::Command::Buffer& command_buffer);
+
+vTensor unpack_image1x1(vTensor v_src, c10::SmallVector<int64_t, 4u> output_sizes, api::Context* context, api::Command::Buffer& command_buffer);
```

Comment:
If these functions are only used in one single implementation file, please remove the common header.  Reason: Software engineering is the art (since it is not all science unfortunately) and science of change management, and the bedrock of managing changes is limiting scope.  Limiting scope in general is the single most important tool software engineers have to get a handle on entropy.